examples/own_dataset/train.py

```diff
@@ -0,0 +1,209 @@
+                           postprocess_label=postprocess_label,
+                           labels=labels, smiles_col='SMILES')
+    dataset = parser.parse(args.datafile)["dataset"]
+
+    if args.scale == 'standardize':
+        # Standard Scaler for labels
+        ss = StandardScaler()
+        labels = ss.fit_transform(dataset.get_datasets()[-1])
+        dataset = NumpyTupleDataset(*dataset.get_datasets()[:-1], labels)
+
+    train_data_size = int(len(dataset) * train_data_ratio)
+    train, val = split_dataset_random(dataset, train_data_size, seed)
+
+    # Network
+    n_unit = args.unit_num
+    conv_layers = args.conv_layers
+    if method == 'nfp':
+        print('Train NFP model...')
+        model = GraphConvPredictor(NFP(out_dim=n_unit, hidden_dim=n_unit,
+                                       n_layers=conv_layers),
+                                   MLP(out_dim=class_num, hidden_dim=n_unit))
+    elif method == 'ggnn':
+        print('Train GGNN model...')
+        model = GraphConvPredictor(GGNN(out_dim=n_unit, hidden_dim=n_unit,
+                                        n_layers=conv_layers),
+                                   MLP(out_dim=class_num, hidden_dim=n_unit))
+    elif method == 'schnet':
+        print('Train SchNet model...')
+        model = GraphConvPredictor(
+            SchNet(out_dim=class_num, hidden_dim=n_unit, n_layers=conv_layers),
+            None)
+    elif method == 'weavenet':
+        print('Train WeaveNet model...')
+        n_atom = 20
+        n_sub_layer = 1
+        weave_channels = [50] * conv_layers
+        model = GraphConvPredictor(
+            WeaveNet(weave_channels=weave_channels, hidden_dim=n_unit,
+                     n_sub_layer=n_sub_layer, n_atom=n_atom),
+            MLP(out_dim=class_num, hidden_dim=n_unit))
+    elif method == 'rsgcn':
+        print('Train RSGCN model...')
+        model = GraphConvPredictor(
+            RSGCN(out_dim=n_unit, hidden_dim=n_unit, n_layers=conv_layers),
+            MLP(out_dim=class_num, hidden_dim=n_unit))
+    else:
+        raise ValueError('[ERROR] Invalid method {}'.format(method))
+
+    train_iter = I.SerialIterator(train, args.batchsize)
+    val_iter = I.SerialIterator(val, args.batchsize,
+                                repeat=False, shuffle=False)
+
+    def scaled_abs_error(x0, x1):
+        if isinstance(x0, Variable):
+            x0 = cuda.to_cpu(x0.data)
+        if isinstance(x1, Variable):
+            x1 = cuda.to_cpu(x1.data)
+        if args.scale == 'standardize':
+            scaled_x0 = ss.inverse_transform(cuda.to_cpu(x0))
+            scaled_x1 = ss.inverse_transform(cuda.to_cpu(x1))
+            diff = scaled_x0 - scaled_x1
+        elif args.scale == 'none':
+            diff = cuda.to_cpu(x0) - cuda.to_cpu(x1)
+        return numpy.mean(numpy.absolute(diff), axis=0)[0]
+
+    classifier = L.Classifier(model, lossfun=F.mean_squared_error,
+                              accfun=scaled_abs_error)
```

Comment:
Could you add a comment somewhere that scaled errors are reported as `main/accuracy` and `validation/main/accuracy`  because using `accfun` for calculating a (scaled) error is not ordinal usage of `Classifier`.